sprint-planning/readme.md

```diff
@@ -1,29 +1,83 @@
 # Planning
 
-## Content TODO's
+## Goals
 
-- Complete outline below
-- Determining team capacity
-- Tasking sprint stories
-  - size of tasks
-- WIP limits / dividing up into workstreams
+During the [sprint planning](https://www.agilealliance.org/glossary/sprint-planning), the team discusses and agrees on the scope for the upcoming sprint.
 
-## Goals
+**Goals are:**
+
+- Selecting the **stories** that will be implemented in the sprint.
+- Estimating the **effort** required for the stories in the sprint.
+- Splitting the stories into **tasks**.
+
+General guidance:
 
-What are the intended outcomes of the ceremonies?
+- The sprint planning should happen at the begining of the new sprint (or at the end of the previous one). It usually last bewteen 1 and 4 hours depending on the size of the team and duration of the sprints.
+- Each story should be achievable in the sprint. Otherwise, you should split the story.
+- Each task duration should be somewhere between 2 to 8 hours.
 
 ## Participation
 
-What members of the team should participate?
+**Everyone** in the team should participate in the sprint planning.
+
+**Specific roles:**
+
+- **[Scrum Master](https://www.agilealliance.org/glossary/scrum-master)**:
+  - Facilitates the conversation.
+  - Ensures everyone is heard.
+  - Reminds scrums principles and sprint planning goals if necessary.
+- **[Product owner](https://www.agilealliance.org/glossary/product-owner/)**:
+  - Prior to the sprint planning: performs some [backlog grooming](../backlog-management/grooming/readme.md) to ensure that each story that they want to propose for the new sprint (*) :
+  
+    - Is in the correct position in the backlog, by right priority order.
+    - Contains a clear description.
+    - Contains a clear [definition of done](https://www.agilealliance.org/glossary/definition-of-done/).
+    - Do NOT pre assigns stories to the future sprint. This is the purpose of the sprint planning.
+  - During the meeting:
+  
+    - Describes to the team the stories that they propose for the sprint.
+
+- **All team members**:
+
+  - Listen to the product owner story description.
+  - Ask questions to make sure everyone understands each story properly.
+  - [Estimate](./estimation/readme.md) the effort for each backlog item, as a team.
+  - Split each story into tasks.
+  - (Optional) self assign first task to team members.

```

Comment:
I am surprised that in planning you would have time to split stories into tasks, usually this is left to whoever owns the user story. 

I have seen stories be assigned to team members at planning vs tasks.

Reply:
First, I would say the team owns the story...the team owns the code...the team owns the sprint. There is no story owner...maybe a lead is assigned, but collectively we own the success of the sprint.

In planning, if there is a high priority story, it may be that the team would need to work together to get it done as it may be a blocker for other stories prioritized for that sprint. Taking some time to task out the story can find opportunities for parallel or dependent work. But you wouldn't necessarily need to task out every story (especially small ones).

Reply:
Agreed with Brian. The classic recommendation is to divide the sprint planning in 2 parts:

- One during which the team build the backlog at the story level
- One during which the team splits stories into tasks, when it make sense.

This ensures the entire team is aware of everything that needs to be done.

This is the typical recommendation. But I am happy to add something here that makes it optional.

Reply:
We have also split the stories into tasks during planning, partially to ensure everyone is on the same page and the acceptance criteria are understood and will be met